src/txgraph.cpp

```diff
@@ -356,14 +428,65 @@ void Cluster::Updated(TxGraphImpl& graph) noexcept
                 DepGraphIndex idx = m_linearization[lin_idx++];
                 GraphIndex graph_idx = m_mapping[idx];
                 auto& entry = graph.m_entries[graph_idx];
-                entry.m_chunk_feerate = FeePerWeight::FromFeeFrac(chunk.feerate);
+                entry.m_main_chunk_feerate = FeePerWeight::FromFeeFrac(chunk.feerate);
                 Assume(chunk.transactions[idx]);
                 chunk.transactions.Reset(idx);
             } while(chunk.transactions.Any());
         }
     }
 }
 
+void Cluster::GetConflicts(const TxGraphImpl& graph, std::vector<Cluster*>& out) const noexcept
+{
+    for (auto i : m_linearization) {
+        auto& entry = graph.m_entries[m_mapping[i]];
+        // For every transaction Entry in this Cluster, if it also exists in a lower-level Cluster,
+        // then that Cluster conflicts.
+        if (entry.m_locator[m_level - 1].IsPresent()) {
+            out.push_back(entry.m_locator[m_level - 1].cluster);
```

Comment:
Should be set to avoid duplicates, will prevent de duplicating later on?

Reply:
Agreed, I calculated recently.
This is better.

The constant factors for GetDistinctClusters implementation using vector and then sorting is also better 👍🏾
Please resolve.